dev-lang/spidermonkey/spidermonkey-140.1.0.ebuild

```diff
@@ -10,7 +10,7 @@ LLVM_COMPAT=( 19 20 )
 RUST_NEEDS_LLVM=1
 RUST_MIN_VER=1.82.0
 
-PYTHON_COMPAT=( python3_{11..13} )
+PYTHON_COMPAT=( python3_{11..14} )
```

Comment:
Let's focus on 140.2.0 and drop this commit. Also, did you test that this works by running the test phase? Because at least firefox-140 won't compile with python-3.14, but I don't remember how "deep" the problem was. Anyway, without testing personally, I'm a bit sceptical about python-3.14 support in SM. If the test phase passes, then it's fine.